sale_stock_picking_return_invoicing/views/stock_picking_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="view_picking_form" model="ir.ui.view">
+    <field name="inherit_id" ref="stock.view_picking_form"/>
+    <field name="model">stock.picking</field>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='owner_id']/.." position="after">
+            <field name="location_usage" invisible="1"/>
+            <field name="to_refund_lines"
+                   attrs="{'invisible': [('location_usage', '!=', 'internal')]}"
```

Comment:
This shouldn't use this condition, but be based on the `picking_type_code` field, when it's equal to `outgoing`.